aten/src/ATen/native/sparse/SparseTensor.cpp

```diff
@@ -376,7 +379,7 @@ SparseTensor coalesce_sparse_cpu(const SparseTensor& self) {
   auto indicesBufferAccessor = indicesBuffer.accessor<int64_t, 1>();
 
   int64_t i = -1;
-  AT_DISPATCH_ALL_TYPES(
+  AT_DISPATCH_ALL_TYPES_AND(ScalarType::Half,
```

Comment:
I'm confused what we are trying to support, because I thought we weren't trying to support addition.  `coalesce()` is slightly different and I think it's okay to support it if you want, but is this intentional?

Reply:
hmm... we don't need the half implementation here .... but since I reverted the `cpu_half: True` from the coalesce section in native_functions.yaml I figured it wasn't actually exposed and was OK to leave it?